crates/nostr/src/event/tag.rs

```diff
@@ -921,6 +931,11 @@ mod tests {
             .as_vec()
         );
 
+        assert_eq!(
+            vec!["lnurl", "https://lnurltest.com/lnurl/payreq/1337"],
```

Comment:
Can you replace this with `lnurl1dp68gurn8ghj7um5v93kketj9ehx2amn9uh8wetvdskkkmn0wahz7mrww4excup0dajx2mrv92x9xp`?